src/js/utils/strings.js

```diff
@@ -4,3 +4,10 @@ export const formatArbitratorName = (user, address, noUsernameLabel, index) => {
   if (!user) return `${index >= 0 ? index + 1 + ' - ' : ""}` + address + ' - Loading...';
   return `${index >= 0 ? index + 1 + ' - ' : ""}${user.username || (noUsernameLabel ? noUsernameLabel : "No username available")} (${compactAddress(address, 3)})  ${user.location ? ' from ' + user.location : ''} - ${user.upCount || 0}↑  ${user.downCount || 0}↓`;
 };
+
+export const getContactDataItem = (contactData, item) => {
```

Comment:
This makes me think, we should probably make a reverse function too where it generates the contact data string with the `:`, that way, the components don't have to know about it, they just give a method and a string.